ocfweb/main/templates/main/home.html

```diff
@@ -3,19 +3,28 @@
 {% load humanize %}
 {% load lab_hours %}
 {% load staff_hours %}
+{% load common %}
 
 {% block hero %}
-<div class="ocf-penguin-swing"></div>
-
 <div class="ocf-hero">
     <div class="container">
         <div class="row">
-            <div class="col-sm-8">
-                <h1>Welcome to the Open Computing Facility!</h1>
+            <div class="col-sm-6">
+                <h1>Welcome to the Open Computing Facility!</h1><br>
                 <p>The Open Computing Facility (OCF) is an all-volunteer student organization dedicated to free computing for all University of California, Berkeley students, faculty, and staff.</p>
-                <p class="ocf-button-holder">
-                <a href="{% url 'about-staff' %}" class="btn btn-lg">Learn what we do</a>
-                <a href="{% url 'register' %}" class="btn btn-lg">Create an account</a>
+            </div>
+            <div class="col-sm-6 ocf-login-main">
+                <a class="sso-button" title="For current students only" id="sso-button" data-bind="click: track_sso_click" href="https://auth.berkeley.edu/cas/login?service=https%3A%2F%2Fwww.ocf.berkeley.edu%2Flogin%2Fcalnet%2Flogin%2F%3Fnext%3D%252Faccount%252Fregister%252F&renew=true">
+                  <div class="sso-content">
+                    <img alt="" src="https://s3.amazonaws.com/handshake.production/app/public/assets/schools/84/small/hs-school-logo-data.?1482461526">
```

Comment:
This image should be hosted by us, from static assets, not from another website.